library/bn_mul.h

```diff
@@ -1006,6 +1005,7 @@
 #endif /* MSVC */
 
 #endif /* MBEDTLS_HAVE_ASM */
+/* *INDENT-ON* */
```

Comment:
Why restart styling here? Uncrustify doesn't particularly help with the rest of the file either.

Reply:
Uncrustify runs without erroring out if asked to restyle the rest of the file. But I'm not fully confident it'll do the right thing. And we might add more problematic assembly later for another architecture. So please keep the rest of the file unstyled.